include/LocalHostStats.h

```diff
@@ -43,7 +43,12 @@ class LocalHostStats: public HostStats {
   struct ndpi_hll hll_contacted_hosts;
   double old_hll_value, new_hll_value, hll_delta_value;
   DESCounter contacted_hosts;
-  
+
+  /* Estimate the number of contacted countries using HyperLogLog */
```

Comment:
Quantify, and report, in terms of Bytes, the impact of all these members for each local hosts.